.github/workflows/checks.yml

```diff
@@ -0,0 +1,41 @@
+name: Checks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  checks:
+    runs-on: ubuntu-20.04
```

Comment:
I would suggest `ubuntu-latest`, and similar in other places were a specific Ubuntu or Python version is set, but I guess it comes down to https://github.com/scrapy/queuelib/pull/48

Reply:
No objections here, this won't change frequently.